client/post-editor/editor-sidebar/index.jsx

```diff
@@ -27,8 +29,31 @@ export default class EditorSidebar extends Component {
 		isPrivate: PropTypes.bool,
 	}
 
+	constructor() {
+		super();
+		this.state = { showFeedback: false };
+	}
+
+	openFeedbackPane = () => {
+		this.setState( { showFeedback: true } );
+	};
+	closeFeedbackPane = () => {
+		this.setState( { showFeedback: false } );
+	};
```

Comment:
I have mixed feelings about this. On one hand, I prefer expressions to imperative statements, but on the other, I don't like returning the value of an expression unless it is explicitly intended. In some cases, the latter can be a problem depending on whether the caller optionally consumes a return value. In this case, I doubt setState returns anything, and React doesn't appear to use the return value of event listeners to do things like preventDefault.

All of that said... I think your sample reads better and will change my code to match.

Thanks